src/ansys/additive/core/parametric_study/parametric_study.py

```diff
@@ -132,6 +132,8 @@ def import_csv_study(self, file_name: str | os.PathLike) -> list[str]:
 
         # The first column of the CSV file is expected to be the index column.
         columns = [getattr(ColumnNames, k) for k in ColumnNames.__dict__ if not k.startswith("_")]
+        columns.remove(ColumnNames.PV_RATIO)
+        # check if all columns in the CSV file are a subset of the expected columns
```

Comment:
It looks to me like this will raise an exception if the csv has the P/V column. See my comment below, we need a test to import a csv containing the P/V column.

Reply:
yes, this does raise an exception. I will re do some of this logic.